src/app/components/RoundedBalance/index.tsx

```diff
@@ -27,7 +26,7 @@ export const RoundedBalance: FC<RoundedBalanceProps> = ({
   compactLargeNumbers,
   scope,
   showSign,
-  ticker,
+  ticker = '',
```

Comment:
Nit: I would prefer if it was `null`.

Reply:
Not sure if that could break something. I'll keep this cleanup without logic changes